api/v1beta1/openstackdataplanenodeset_types.go

```diff
@@ -67,40 +65,51 @@ type OpenStackDataPlaneRoleSpec struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
-//+operator-sdk:csv:customresourcedefinitions:displayName="OpenStack Data Plane Role"
-// +kubebuilder:resource:shortName=osdprole;osdproles
+//+operator-sdk:csv:customresourcedefinitions:displayName="OpenStack Data Plane NodeSet"
+//+kubebuilder:resource:shortName=osdpns;osdpnodeset;osdpnodesets
 //+kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.conditions[0].status",description="Status"
 //+kubebuilder:printcolumn:name="Message",type="string",JSONPath=".status.conditions[0].message",description="Message"
 
-// OpenStackDataPlaneRole is the Schema for the openstackdataplaneroles API
-type OpenStackDataPlaneRole struct {
+// OpenStackDataPlaneNodeSet is the Schema for the openstackdataplanenodesets API
+type OpenStackDataPlaneNodeSet struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   OpenStackDataPlaneRoleSpec `json:"spec,omitempty"`
-	Status OpenStackDataPlaneStatus   `json:"status,omitempty"`
+	Spec   OpenStackDataPlaneNodeSetSpec `json:"spec,omitempty"`
+	Status OpenStackDataPlaneNodeSetStatus      `json:"status,omitempty"`
+}
+
+// OpenStackDataPlaneNodeSetStatus defines the observed state of OpenStackDataPlaneNodeSet
+type OpenStackDataPlaneNodeSetStatus struct {
+	// +operator-sdk:csv:customresourcedefinitions:type=status,xDescriptors={"urn:alm:descriptor:io.kubernetes.conditions"}
+	// Conditions
+	Conditions condition.Conditions `json:"conditions,omitempty" optional:"true"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=status,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
+	// Deployed
+	Deployed bool `json:"deployed,omitempty" optional:"true"`
 }
 
 //+kubebuilder:object:root=true
 
-// OpenStackDataPlaneRoleList contains a list of OpenStackDataPlaneRole
-type OpenStackDataPlaneRoleList struct {
+// OpenStackDataPlaneNodeSetList contains a list of OpenStackDataPlaneNodeSets
+type OpenStackDataPlaneNodeSetList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty"`
-	Items           []OpenStackDataPlaneRole `json:"items"`
+	Items           []OpenStackDataPlaneNodeSet `json:"items"`
 }
 
 func init() {
-	SchemeBuilder.Register(&OpenStackDataPlaneRole{}, &OpenStackDataPlaneRoleList{})
+	SchemeBuilder.Register(&OpenStackDataPlaneNodeSet{}, &OpenStackDataPlaneNodeSetList{})
 }
 
 // IsReady - returns true if the DataPlane is ready
-func (instance OpenStackDataPlaneRole) IsReady() bool {
-	return instance.Status.Conditions.IsTrue(condition.ReadyCondition)
+func (instance OpenStackDataPlaneNodeSet) IsReady() bool {
+	return instance.Status.Conditions.IsTrue(condition.DeploymentReadyCondition)
```

Comment:
This is undoing part of your commit to fix conditions isn't it? e40961c9a99343422671ada69c0ddcb4e2a6101c
Is this a bad rebase?

Reply:
fixed it